modules/kubernetes-addons/helm_addon/main.tf

```diff
@@ -0,0 +1,72 @@
+resource "helm_release" "addon" {
+  count                      = var.manage_via_gitops ? 0 : 1
+  name                       = var.helm_config["name"]
+  repository                 = var.helm_config["repository"]
+  chart                      = var.helm_config["chart"]
+  version                    = var.helm_config["version"]
+  timeout                    = var.helm_config["timeout"]
```

Comment:
Helm default timeout is 1200. I think its better set defaults as 300 for all add-ons unless user overrides

```
timeout        = try(var.helm_config["timeout"], 300)
```

Reply:
addressed